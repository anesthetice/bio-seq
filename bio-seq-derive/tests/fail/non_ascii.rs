use bio_seq::prelude::*;

fn main() {
    let _seq = iupac!("ACNSWGTğŸ¤¬");
    let _seq = dna!("ACGğŸ«£T");
}
