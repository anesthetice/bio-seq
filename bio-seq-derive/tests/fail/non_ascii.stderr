error: Non-ASCII characters in IUPAC string
 --> tests/fail/non_ascii.rs:4:23
  |
4 |     let _seq = iupac!("ACNSWGTğŸ¤¬");
  |                       ^^^^^^^^^^^

error: Non-ASCII characters in DNA string
 --> tests/fail/non_ascii.rs:5:21
  |
5 |     let _seq = dna!("ACGğŸ«£T");
  |                     ^^^^^^^^
